packages/manager/modules/pci/src/projects/project/streams/stream/subscriptions/delete/delete.routing.js

```diff
@@ -0,0 +1,22 @@
+export default /* @ngInject */($stateProvider) => {
+  $stateProvider.state('pci.projects.project.streams.stream.subscriptions.delete', {
+    url: '/delete?subscriptionId',
+    views: {
+      modal: {
+        component: 'ovhManagerPciStreamsStreamSubscriptionsDelete',
+      },
+    },
+    layout: 'modal',
+    resolve: {
+      goBack: /* @ngInject */ goToSubscriptions => goToSubscriptions,
+      breadcrumb: () => null,
```

Comment:
Why do we set it to `null` if it's unused in the component ? (Same for the others modals)

Reply:
If we don't define `breadcrumb: () => null` it will inherit from parent resolve.
https://github.com/ovh-ux/manager/blob/master/packages/components/ng-ui-router-breadcrumb/src/ng-ui-router-breadcrumb.service.js#L24

Reply:
Ok :)